pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml

```diff
@@ -0,0 +1,15 @@
+# This YAML file contains configmap objects,
+# which are necessary to run csi cinder plugin.
+
+kind: ConfigMap
```

Comment:
This has to be secret as it contains user name & password.

Reply:
+1

Reply:
Done